koku/api/report/gcp/query_handler.py

```diff
@@ -37,8 +37,16 @@ class GCPReportQueryHandler(ReportQueryHandler):
     """Handles report queries and responses for GCP."""
 
     provider = Provider.PROVIDER_GCP
-    network_services = {}  # FIXME: When we start working on networking apis
-    database_services = {}  # FIXME: When we start working on database apis
+
+    network_services = {
```

Comment:
Will want to match the same strings that are in the materialized view and have the UI update to query with those strings.